src/shared/components/annotation/Gistic.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+import Icon from 'shared/components/cohort/LetterIcon';
+import { If, Then, Else } from 'react-if';
+
+/**
+ * Gets the GISTIC ToolTip Component.
+ * @param qValue GISTIC Q-Vale.
+ * @param peakGeneCount Number of Genes within the GISTIC Peak.
+ */
+export function getGisticTooltip(qValue: number, peakGeneCount = 0) {
+    return (
+        <div>
+            Candidate driver gene, as determined by the <b>GISTIC</b> algorithm.
+            <p />
+            The GISTIC algorithm identifies likely driver copy number
+            <br />
+            alterations by evaluating the frequency and amplitude of
+            <br />
+            observed CNA events within the entire study.
+            <p />
+            <span> Q-value: {(qValue || 0).toExponential(3)}</span>
+            <If condition={peakGeneCount > 0}>
+                <Then>
+                    <br />
+                    <span> Number of genes in the peak: {peakGeneCount}</span>
+                </Then>
+            </If>
+        </div>
+    );
+}
+
+/**
+ * Makes the GISTIC Icon
+ * @param qValue GISTIC Q-Value
+ * @param peakGeneCount Number of Genes within the GISTIC Peak
+ */
+export function makeGisticIcon(qValue: number, peakGeneCount: number) {

```

Comment:
How about `Gistic` as a proper react component instead of this function? 

It was okay to have this as a static function inside `CohortColumnFormatter`, but since we already moved the code out of the formatter, it is better to have a proper react component here. You can define `qValue` and `peakGeneCount` as `props` of the `Gistic` component.

Reply:
Thanks, Onur!  And, great suggestion.

Before I do so though, can you clarify one issue?

Right now, the patient view and study view pages actually use different mechanisms for displaying the (M) and (G) icons.  For patient view, we use the LetterIcon component and for the study view page, we use PNG icons, e.g. https://github.com/cBioPortal/cbioportal-frontend/tree/master/src/pages/studyView/table/images.

I didn't want to break the latter, thinking there was probably some good reason for this and I also didn't want to break all the study view screenshot tests.  Do you think it's ok to delete the second mechanism, with the PNG icons?   And, then I need to fix all the screenshot tests?

Reply:
@ecerami I think it is a good idea to replace the study view PNG icons with this one as well. We might have introduced the `LetterIcon` component after implementing the study view tables, or we might have simply missed the `LetterIcon` and added PNGs instead.

Right, you may need to fix the tests if we have the (G) and (M) icons in the reference screenshots.

Reply:
@onursumer I pushed a new commit where I created true React components for MutSig and GISTIC annotation.  I also deleted the PNG icons.  Take a look and let me know what you think.  If it looks good, I will go ahead and squash the 6 commits into one.